rke/k8s_rke_system_images.go

```diff
@@ -2438,6 +2474,41 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  m("rancher/kubelet-pause:v0.1.3"),
 			Nodelocal:                 m("k8s.gcr.io/k8s-dns-node-cache:1.15.7"),
 		},
+		// Enabled out of band for Rancher v2.3.8
```

Comment:
Can this be updated to: `// Enabled out of band for Rancher v2.3.8 with Rancher v2.4.5`?